src/modules/openrcdmcryptcfg/main.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# === This file is part of Calamares - <http://github.com/calamares> ===
+#
+#   Copyright 2017, Ghiunhan Mamut <venerix@redcorelinux.org>
+#
+#   Calamares is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   Calamares is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with Calamares. If not, see <http://www.gnu.org/licenses/>.
+
+import libcalamares
+import os.path
+
+
+def write_dmcrypt_conf(partitions, root_mount_point, dmcrypt_conf_path):
+    crypto_target = ""
+    crypto_source = ""
+
+    for partition in partitions:
+        if partition["mountPoint"] == "/home" \
+                and "luksMapperName" in partition:
+            crypto_target = partition["luksMapperName"]
+            crypto_source = "/dev/disk/by-uuid/{!s}".format(partition["uuid"])
+    if "luksMapperName" not in partition:
+            return None
+
+    with open(os.path.join(root_mount_point,
+                           dmcrypt_conf_path),
+                           'a+'
```

Comment:
```
/home/travis/build/calamares/calamares/src/modules/openrcdmcryptcfg/main.py:39:28: E127 continuation line over-indented for visual indent
/home/travis/build/calamares/calamares/src/modules/openrcdmcryptcfg/main.py:40:28: E124 closing bracket does not match visual indentation
```
Maybe I was wrong on my end. My suggestion would be this:

```
    with open(os.path.join(root_mount_point,
                           dmcrypt_conf_path
                           ), 'a+') as dmcrypt_file:
```
@adriaandegroot: correct me if I'm again wrong here ...